Jenkinsfile

```diff
@@ -0,0 +1,129 @@
+@Library('github.com/glogiotatidis/jenkins-pipeline@more-commands')
```

Comment:
Just merged https://github.com/mozmar/jenkins-pipeline/pull/2 so you should be able to update this to use it now.